activesupport/lib/active_support/core_ext/object/inclusion.rb

```diff
@@ -1,11 +1,11 @@
 class Object
-  # Returns true if this object is included in the argument. Argument must be
+  # Returns +true+ if this object is included in the argument. Argument must be
```

Comment:
Actually we'd only want to mark true like that if the return value is `true` and not just truthy. So revert this change.